cosmos-gui/conf/cosmos-gui.json

```diff
@@ -0,0 +1,22 @@
+{
+  "gui": {
+    "port": 80
+  },
+  "hdfs": {
+    "quota": 5
+  },
+  "oauth2": {
+    "idmURL": "",
+    "client_id": "",
+    "client_secret": "",
+    "callbackURL": "http://localhost:80/auth",
+    "response_type": "code"
+  },
+  "mysql": {
+    "host": "",
+    "port": 3306,
+    "user": "",
+    "password": "",
+    "database": "cosmos_gui"
+  }
+}
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/a72e21d03b37b63e9e8feb6c85c9e9ebef39d63c